docs/training-debug/index.txt

```diff
@@ -1,343 +1,395 @@
+   If you customized your command environment in testing :ref:`Step 3 <step3>`, :ref:`Step 4
+   <step4>`, or :ref:`Step 5 <step5>`, make sure to apply the same customizations in your experiment
+   configuration file.
 
--  The ``checkpoint_storage`` settings are used for cluster-managed training. If
-   ``checkpoint_storage`` was configured in neither the experiment config nor the master config, you
-   will see an error message during experiment config validation, before the experiment or any
-   trials are created. To correct it, simply provide a ``checkpoint_storage`` configuration in one
-   of those locations (:ref:`master-configuration` or :ref:`experiment-configuration`).
+#. Set ``resources.slots_per_trial``:
 
--  The configured ``checkpoint_storage`` settings are validated before training starts for an
-   experiment on the cluster. If you get a message saying ``Checkpoint storage validation failed``,
-   please review the correctness of the values in your ``checkpoint_storage`` settings.
+   Confirm that your experiment config does not specify ``resources.slots_per_trial`` or that it is
+   set to ``1``. For example:
 
--  The experiment config is fully validated for cluster-managed experiments, more strictly than it
-   is for ``--local --test`` mode. If you get errors related to ``unmarshaling JSON`` when trying to
-   submit the experiment to the cluster, that is an indication that the experiment config has
-   errors. Please review the :ref:`experiment configuration <experiment-configuration>`.
+   .. code:: yaml
 
-Again, if you are unable to identify the root cause of the issue yourself, please do not hesitate to
-contact Determined through our `community support
+      resources:
+      slots_per_trial: 1
+
+#. Create an experiment with the ``--test`` argument, omitting the ``--local`` argument:
+
+   .. code:: bash
+
+      det experiment create myconfig.yaml my_model_dir --test
+
+#. Perform failure diagnostics:
+
+   If you can run local test mode inside a notebook or shell but are unable to successfully submit
+   an experiment, make sure that notebook or shell customizations you might have made are replicated
+   in your :ref:`experiment configuration <experiment-configuration>`, such as:
+
+   -  If required, a custom Docker image is set in the experiment configuration.
+
+   -  ``pip install`` or ``apt install`` commands needed in the interactive environment are built
+      into a custom Docker image or included in the ``startup-hook.sh`` file in the model definition
+      directory root. See :ref:`startup-hooks` for more information.
+
+   -  Custom bind mounts required in the interactive environment are specified in the experiment
+      configuration.
+
+   -  Environment variables are correctly set in the experiment configuration.
+
+   If no customizations are missing, the following, new layers introduced with a cluster-managed
+   experiment could be the problem cause:
```

Comment:
"could be the cause of the problem" feels far more idiomatic to me than "the problem cause"

Reply:
Already fixed.